src/pages/Dashboards/dashboards/SocialTool/hooks.js

```diff
@@ -1,15 +1,12 @@
 import { useTrendingWords, useTrendWordsData } from '../../../../ducks/TrendsTable/hooks'
 import { WORDS_SOCIAL_DOMINANCE_QUERY } from '../../../../ducks/TrendsTable/queries'
-import { getSocialDominanceSum } from '../../../../ducks/TrendsTable/utils'
 
 export function useSocialDominanceTrendingWords(period) {
   const { words, isLoading } = useTrendingWords(period)
-  const data = useTrendWordsData(WORDS_SOCIAL_DOMINANCE_QUERY, words)
-  const wordsDominance = data && Object.keys(data).map((word) => ({ word, value: data[word] }))
-  const socialDominance = wordsDominance && getSocialDominanceSum(wordsDominance)
+  const wordsDominance = useTrendWordsData(WORDS_SOCIAL_DOMINANCE_QUERY, words)
 
   return {
-    data: socialDominance || 0,
+    dominance: wordsDominance || [],
```

Comment:
@Woafflation Are you sure that default should be `[]`? It seems like on `LOC 39 of src/ducks/TrendsTable/columns.js` it's accessed using string key, hence it should be `{}`